.github/workflows/workflows.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: workflows
+name: cloudworkflows
```

Comment:
ooh,  good catch to make this more clear.

Reply:
This was triggering GHA lint errors in vscode.